tests/testthat/test-new-model.R

```diff
@@ -137,7 +137,7 @@ withr::with_options(list(rbabylon.model_directory = NULL), {
     expect_error(
       suppressSpecificWarning({
```

Comment:
this `suppressSpecificWarning` should go away. I'm pretty sure we missed it earlier because it's wrapped in an `expect_error` (that still correctly errors).